README.md

```diff
@@ -1,5 +1,11 @@
 # Yak
 
+[![GitHub Actions status](https://github.com/ros-industrial/yak/workflows/CI/badge.svg?branch=devel)](http://build.ros.org/job/Kdev__universal_robot__ubuntu_xenial_amd64)
```

Comment:
This says "Github actions", but then links to the ROS buildfarm.

Is this correct?

Reply:
Sorry, I updated